tests/by-util/test_cp.rs

```diff
@@ -1349,6 +1349,170 @@ fn test_cp_preserve_xattr_fails_on_android() {
         .fails();
 }
 
+#[test]
+#[cfg(not(target_os = "android"))]
```

Comment:
The failing android tests are not due to this change I believe. Is it really necessary to add this `cfg`?

Don't worry about android, those tests are failing on many PRs and seem to be some android configuration issue.